server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -453,6 +460,29 @@ private boolean canReserveCompaction(TabletMetadata tablet, CompactionJob job,
     return true;
   }
 
+  private void checkTabletDir(KeyExtent extent, Path path) {
+    try {
+      ;
+      if (CHECKED_TABLET_DIRS.getIfPresent(path) != 1) {
```

Comment:
an early return on the inverse of this condition might make this section a bit more readable.

Reply:
this check is incorrect anyway, I'm changing it